site/_docs/history.md

```diff
@@ -56,20 +132,221 @@ other software versions as specified in gradle.properties.
+  Update BigQuery Conformance for != and % operators
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5466">CALCITE-5466</a>]
+  Constant condition can't be reduced after correlate
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5471">CALCITE-5471</a>]
+  RelSupplier.SqlRelSupplier#apply should use .project(), not .rel
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5489">CALCITE-5489</a>]
+  When creating a RexCall to TIMESTAMP_DIFF function, cannot convert a TIMESTAMP literal to a org.apache.calcite.avatica.util.TimeUnit
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5491">CALCITE-5491</a>]
+  Allow TIME and DATE to be args for TIMESTAMPDIFF
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5493">CALCITE-5493</a>]
+  Time zone tests in SqlFunctions should pass in Europe/London
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5495">CALCITE-5495</a>]
+  Allow WEEK and QUARTER in INTERVAL literals
+
 #### Build and test suite
 {: #build-1-33-0}
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5197">CALCITE-5197</a>]
+  Bump gradle to 7.4.2 and add checksum autoupdate
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5306">CALCITE-5306</a>]
+  Remove JDK15/16 EOL non-LTS JDKs in CI
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5340">CALCITE-5340</a>]
+  Tests should fail when actual and expected XML reference files are not identical
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5399">CALCITE-5399</a>]
+  Proj4J license is not compatible with Apache
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5417">CALCITE-5417</a>]
+  Include Proj4J as an api dependency once the license allows it
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5427">CALCITE-5427</a>]
+  Provide code quality/coverage metrics with SonarCloud and JaCoCo
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5428">CALCITE-5428</a>]
+  Reduce minimum Guava version to 16.0.1
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5433">CALCITE-5433</a>]
+  Druid tests hang/fail intermittently in CI
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5474">CALCITE-5474</a>]
+  Disable Sonar quality gates to avoid checks appearing as failures 
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5475">CALCITE-5475</a>]
+  Improve test coverage accuracy by aggregating modules
 
 #### Web site and documentation
 {: #site-1-33-0}
 
-{% endcomment %}
--->
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5239">CALCITE-5239</a>]
+  Site: JDBC Adapter's current limitations is incorrect
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5287">CALCITE-5287</a>]
+  SQL reference page is missing from website
+* Site: Add Benchao Li as PMC
+* Site: Add Istvan Toth as committer
+* Site: Switch PMC Chair to Stamatis Zampetakis
+* Site: Add Bertil Chapuis as committer
+* Site: Add Dmitry Sysolyatin as committer
```

Comment:
Member changes are not that interesting in terms of release notes so they could be omitted.

Reply:
removed